sdks/java/core/src/main/java/org/apache/beam/sdk/io/Compression.java

```diff
@@ -152,6 +156,38 @@ public WritableByteChannel writeCompressed(WritableByteChannel channel) throws I
     }
   },
 
+  /** LZO compression. */
```

Comment:
It may not be obvious to users that the difference between LZO and LZOP is that one has headers and one is just the LZO algorithm. Please expand on this comment and the one below.

Reply:
Agreed. I have added comments for both the modules (LZO, LZOP). This should make things clear.